### PR TITLE
add new notification modal layouts and error handling

### LINKED ISSUE #273: Refactoring | better handling of error messages
## Description
Currently, we are showing every unknown error for the user to handle, but sometimes the error message is too large to display on the screen. We need to try adding the current solution so it shows better feedback to the user while dumping on the console all the error information in order to be available if the user knows how to explore the console.

- make scrollable area for json
- treat already mapped common errors more friendly but have this behavior as default (1 example would be enough for now)
- remove countdown for hiding error message, i.e. require user to manually dismiss it

## Screenshot
image

### PR DESCRIPTION
**Pull request by Viterbo**
# Fixes #273 and #363

## Description 
This PR updates the set of notification functions to call different layouts. Now we can notify success, failure, and neutral messages. Only error messages remain until dismissed manually. The rest of the models decrease the header color line as time passes as if it was a progressive bar and then it closes itself.

Also, a mapping between a list of code errors and a more appropriate text was added in order to give better feedback to the user.

```typescript
switch (evmErr.code) {
    case 'CALL_EXCEPTION':          return 'antelope.evm.error_call_exception';
    case 'INSUFFICIENT_FUNDS':      return 'antelope.evm.error_insufficient_funds';
    case 'MISSING_NEW':             return 'antelope.evm.error_missing_new';
    case 'NONCE_EXPIRED':           return 'antelope.evm.error_nonce_expired';
    case 'NUMERIC_FAULT':           return 'antelope.evm.error_numeric_fault';
    case 'REPLACEMENT_UNDERPRICED': return 'antelope.evm.error_replacement_underpriced';
    case 'TRANSACTION_REPLACED':    return 'antelope.evm.error_transaction_replaced';
    case 'UNPREDICTABLE_GAS_LIMIT': return 'antelope.evm.error_unpredictable_gas_limit';
    case 'USER_REJECTED':           return 'antelope.evm.error_user_rejected';
    case 'ACTION_REJECTED':         return 'antelope.evm.error_transaction_canceled';
}
```

## Demo Page

I added a Demo page with the same Send Page component changed to allow wrong parameters on the form so we can provoke an error.
image

When you get an error with payload information you will see the "See Details" button to display this modal showing the content of the payload.
image

Also, you can try out all the different notifications we have in the notification showroom
image

## Checklist:

-   [ ] I have performed a self-review of my own code
-   [ ] I have commented my code, particularly in hard-to-understand areas
-   [ ] I have cleaned up the code in the areas my change touches
-   [ ] My changes generate no new warnings
-   [ ] Any dependent changes have been merged and published in downstream modules
-   [ ] I have checked my code and corrected any misspellings
-   [ ] I have removed any unnecessary console messages
-   [ ] I have included all english text to the translation file
-   [ ] I have created a new issue with the required translations for the currently supported languages
-   [ ] I have added appropriate test coverage

### CODE REVIEW

**Review by donnyquixotic — APPROVED** [review]

File: src/i18n/en-us/index.js [hunk 1]
```diff
@@ -117,11 +117,22 @@ export default {
         amount: 'Amount',
     },
     notification:{
-        success_title: 'Success',
-        success_message: 'Your transaction has been processed. For detailed information, click the link below.',
+        success_title_trx: 'Success',
+        success_title_copied: 'Copied',
+        success_message_trx: 'Your transaction has been processed. For detailed information, click the link below.',
+        success_message_revoking: 'Revoking <b>{symbol}</b> allowance for <b>{address}</b> was successful.',
+        success_message_copied: 'Your account name has been copied to the clipboard.',
```
Comment (EJSG) [inline src/i18n/en-us/index.js:124]:
```suggestion
        success_message_copied: 'Your account address has been copied to the clipboard.',
```

**Review by EJSG — APPROVED** [review]
this is awesome, one small comment

**Review by karynemayer — COMMENTED** [review]
Good job! I left non-blocking comments, but it looks way better than what we had!

**Comment by Viterbo** [comment]
I'm closing this PR because I created a [new one](https://github.com/telosnetwork/telos-wallet/pull/403) including this one.

File: src/boot/errorHandling.js [hunk 2]
```diff
@@ -67,68 +69,180 @@ const html = `
...
+            let content = '';
+            try {
+                content = JSON.stringify(payload, null, 2);
+            } catch (e) {
+                try {
+                    content = payload.toString();
+                } catch (e) {
+                    content = payload + ' ';
+                }
+            }
```
[the remark this answers is not in the document]
Comment (Viterbo) [reply src/boot/errorHandling.js:103]: I also did address this [on the new PR](https://github.com/telosnetwork/telos-wallet/pull/403/files#r1220544210)

**Pull request closed without merging**